### PR TITLE
Make 18F engineering team CODEOWNER for engineering guide

### PR DESCRIPTION
**Pull request by alexsobledotgov**
## Why?

18F engineering publicly renewed our commitment to investing in our engineering practices, and regularly publishing what we learn. We created a new principal consulting engineer role for this reason. 

This proposed change will help principal engineers + eng supervisors more regularly deliver updates to 18F's public engineering guides.

## What does this PR do?

Gives @18f/18f-engineering-leadership the ability to review and approve PRs that change content in the engineering guide.

## Background

+ In the past, @loganmeetsworld discussed this change with @hbillings who agreed with the idea, but the change was never made.
+ More recently,  principal engineers and engineering supervisors discussed this in our regular meeting and felt this would be a good change to make to our process.

## ⚠️ Before merge

+ Before we merge this PR, @adunkman will need to give the @18f/18f-engineering-leadership write access to the guides repository
+  Via GitHub docs: "Teams must have explicit write access to the repository."
+ The CODEOWNERS on this branch is invalid right now because @18f/18f-engineering-leadership does not have write access. 

## Key reviewer / approver 

@alexbielen, acting for Heather 

## Security considerations

None

### CODE REVIEW

File: .github/CODEOWNERS
Comment (mgwalker) [inline .github/CODEOWNERS:29]: Not strictly an issue with this PR, I think, but GitHub is highlighting this line as an "unknown owner" and makes three recommendations. Check that:

- the team exists (it does)
- it is publicly visible (presumably this is only for the error checking but maybe not!)
- it has write access to the repo (I cannot check)

Pinging @adunkman for visibility.

**Review by mgwalker — APPROVED** [review]
Nice!

File: .github/CODEOWNERS
Comment (alexsobledotgov) [reply .github/CODEOWNERS:29]: Yep! See the "⚠️ Before merge" section in the PR

**Comment by alexsobledotgov** [comment]
@alexbielen plans to discuss at next week's eng leadership meeting

**Comment by cantsin** [comment]
Looks like we have a PR conflict with the changed methods code owner.

**Comment by alexsobledotgov** [comment]
@alexbielen @cantsin PR conflict addressed

**Review by alexbielen — APPROVED** [review]
We discussed this in the last Engineering Leadership meeting. This change is good assuming we follow the process outlined below and recognize that @hbillings may want to do this differently when she returns from leave. 

Process (thanks to @alexsobledotgov for proposing this!):

Proposal for how we combine agility + accountability on Engineering Guide changes:
Bring proposed changes to Guide to engineering leadership meetings for discussion & visibility.

- Tag the engineering director for reviews on PRs.
- We are all adults here and we are all engineering leaders! If a proposed change has been brought to the group for discussion, there are no concerns or red flags, a PR has 👍  from this group all around, and the director isn't able to review — let's say they are on leave and/or eating barbecue — any member of [#18f-eng-leadership](https://gsa-tts.slack.com/archives/C03S933U9) is empowered to approve and merge.
- Make sure the director sees the change when they get back online, ask if they have any questions or proposed follow-on edits.
- The idea here is to make sure the director has visibility into changes and can speak to the content of the guide — but that work doesn't come to a halt when the director gets pulled in other directions.

**Comment by alexsobledotgov** [comment]
Great, thanks @alexbielen! 

@adunkman, at your earliest availability, could you give the @18F/18f-engineering-leadership team write access to the guides repository and merge the PR? 

Thank you!

**Comment by adunkman** [comment]
Completed! I’ve also removed specific named access to Alex and Heather, as they now inherit access from the new team. Give me a shout if that broke anything!

**Pull request merged**
